docs/topics/intellij-idea-plugin-run-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-run-configuration_{context}"]
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project you import to IntelliJ IDEA.
+
+.Procedure
+
+. In *Projects* view, click the project you want to analyze.
+. On the left side of the screen, click the *{ProductName}* tab.
++
+If this is your first configuration, the run configuration panel is displayed on the right.
+
+. If this is not your first configuration, right-click on one of the configurations in the list and select *New configuration*.
++
+The run configuration panel is displayed on the right.
+
+. Complete the following configuration fields:
+
+** *mta-cli*: Click *Add*, select the filepath, and click *Save*.
+** *Input*: Click *Add* and do one of the following:
+
+*** Enter the input file or directory and press Enter.
+*** Click *Open File Explorer*, select the directory, and click *OK*.
+
+** *Target*: Select one or more target migration paths.
++
+[NOTE]
+====
+The location shown in the *Output* is set by the plugin.
```

Comment:
Passive voice.
```suggestion
The plugin sets the location shown in the *Output* field.
```

Reply:
The IBM Style Guide says that the passive voice is acceptable when the system performs the action (so, no change).
![passive](https://user-images.githubusercontent.com/60698649/155504764-7fd81163-abc6-4055-8688-20a8f73c716a.png)

Reply:
OK, thanks for showing me that!